config/deployment.yaml

```diff
@@ -52,6 +52,9 @@ spec:
         volumeMounts:
         - name: fulcio-config
           mountPath: /etc/fulcio-config
+        - name: fulcio-secret
+          mountPath: /etc/fulcio-secret
+          readOnly: true
```

Comment:
I believe this will cause production to break, but I believe you can make it optional (and then it won't)

Reply:
Does the `optional` field added in `volumes` section below accomplish this?

Reply:
I think so. Tried adding optional in another branch and got yelled at:
```
error: error validating "STDIN": error validating data: ValidationError(Deployment.spec.template.spec.containers[0].volumeMounts[1]): unknown field "optional" in
```